app/views/users/singlepost.html.erb

```diff
@@ -0,0 +1 @@
+<h1>This is a single post page</h1>
```

Comment:
- [ ] We don't need this file. Kindly remove it.

- [ ] Instead of this we can create a `app/views/users/show.html.erb` file to handle the views from the `show` method defined in the Users controller.